docs/02_cleos/03_command-reference/net/connect.md

```diff
@@ -1,17 +1,56 @@
+## Command
+```sh
+cleos net connect [OPTIONS] host
+```
+
+**Where:**
+* [OPTIONS] = See **Options** in the [**Command Usage**](command-usage) section below.
+* host = The hostname:port to connect to.
+
+**Note:** The arguments and options enclosed in square brackets are optional.
+
 ## Description
-Start a new connection to a peer
+Start a new connection to a specified peer. This command allows an operator's node to connect to another peer without restarting the node.
+
+## Command Usage
+The following information shows the different positionals and options you can use with the `cleos net connect` command:
+
+### Positionals
+* `host` _TEXT_ REQUIRED - The hostname:port to connect to.
+
+### Options
+* `-h,--help` - Print this help message and exit.
+
+## Requirements
+Make sure you meet the following requirements:
 
-**Command**
+* Install the currently supported version of `cleos`.
+[[info | Note]]
+| `cleos` is bundled with the EOSIO software. [Installing EOSIO](../../../00_install/index.md) will also install the `cleos` and `keosd` command line tools.
+* You have access to a producing node instance with the [`net_api_plugin`](../../../01_nodeos/03_plugins/net_api_plugin/index.md) loaded.
 
+## Examples
+The following examples demonstrate how to use the `cleos net connect` command:
+
+* Instruct default local node (listening at default http address `http://127.0.0.1:8888`) to connect to peer node listening at p2p address `localhost:9022`:
```

Comment:
Should the examples be numbered rather than bulleted?

Reply:
I thought bullets were used when sequence doesn't matter. If we want to identify the examples, then I think we should make it explicit; e.g. Example 1, Example 2, etc. because numbered bullets in a list are used for a different purpose. @bobgt ?

Reply:
This is a good scenario. 

If we use numbered list here, it may give a quick illusion that it's a procedure. So numbered list won't be a good fit. 

If we go with bullets, it's not inappropriate but the 2 examples are not standing out clearly. 

I would go with the explicit subtitles, L3 Headers:

### Example 1 
Instruct default local node (listening at default http address `http://127.0.0.1:8888`) to connect to peer node listening at p2p address `localhost:9022`:
```sh
cleos net connect localhost:9022
```

**Output:**

```console
"added connection"
```
### Example 2 

I will consult with Virginia and update the Command Reference template. Thanks for flagging this!